src/cascadia/TerminalCore/TerminalApi.cpp

```diff
@@ -426,6 +426,18 @@ bool Terminal::SetCursorStyle(const DispatchTypes::CursorStyle cursorStyle) noex
     return true;
 }
 
+bool Terminal::RestoreCursorStyleToUserDefault() noexcept
+{
+    if (_buffer)
+    {
+        _buffer->GetCursor().SetStyle(_defaultCursorHeight,
+                                      _defaultCursorColor,
+                                      _defaultCursorShape);
+    }

```

Comment:
Why do we need a new method for this? Wouldn't it be easier to just add it to the switch statement in the existing `SetCursorStyle` method? That also raises the question of why this is resetting values that the `SetCursorStyle` method never changed. I would have thought the only thing it should be changing is the type and blink status?